util/formatter/formatter.go

```diff
@@ -0,0 +1,65 @@
+package formatter
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/docker/cli/templates"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+)
+
+// Format keys used to specify certain kinds of output formats
+const (
+	FormatTable = "table"
+	FormatJSON  = "json"
+)
+
+// Print prints formatted output in different formats
+func Print(o interface{}, format string, w io.Writer, t func(t *tablewriter.Table)) error {
```

Comment:
This does not look consistent with docker cli. There is no `--format json` but `{{json .}}`. And we don't import this "tablewriter" package.